src/software/jetson_nano/display/screens/screen.py

```diff
@@ -0,0 +1,172 @@
+from PIL import ImageFont
+
+import software.jetson_nano.display.constants as constants
+from software.jetson_nano.display.utils import get_ip_address, get_signal_strength
+
+SCREEN_TYPE = str
```

Comment:
curious what this is? can u maybe add a comment explaining it?

Reply:
When displaying the actions on the screen, most of the items have a 'value' field but when changing screens there is no value field. This was a way to identify if the current action was for changing screens and then we would skip displaying the value. I added a comment and changed the if statement for this check so that it may be a bit more clear. 
![image](https://user-images.githubusercontent.com/56134891/161594154-810be7be-9e0f-4251-8191-572fbd8e0bf7.png)